### ISSUE TITLE
Vision synchronizers with GLOBAL_CALL = True use incorrect URL

### ISSUE THREAD

**Issue by caktus-philip** [unicef/etools#1098]
**TLDR**: vision synchronizers construct a URL and then don't use it; the URL that matters is wrong when GLOBAL_CALL = True. Fortunately, no active synchronizers use GLOBAL_CALL = True.

## Long version 

All vision synchronizers inherit from `VisionDataSynchronizer` which sets `self.GLOBAL_CALL = False` by default. Only two synchronizers override this setting (`TravelAgenciesSynchronizer` and `CurrencySynchronizer`). They're not used, which is good, because they would probably fail.

The `VisionDataSynchronizer` initialization includes this code when constructing its URL --

```
if not self.GLOBAL_CALL:
    self.url += '/{}'.format(country.business_area_code)
```

That's fine, but misleading because the synchronizer's URL isn't actually used anywhere. The URL that really matters is on the loader that the synchronizer uses. The loader is a `VisionDataLoader` instance or a subclass of it, and none of the implementations contain that nuance about URL construction. 

**There are two problems identified here.** First is that the synchronizers construct (and log!) a misleading URL and then never use it. The second is that `VisionDataLoader` doesn't correctly construct URLs for synchronizers where `GLOBAL_CALL` is `True`.

The first problem will be fixed in an upcoming PR from me, and I'll update this ticket with a reference to that PR once it exists. I have no plans to address the second problem.
